src/test/cli.ts

```diff
@@ -44,6 +44,7 @@ export function addTestCommand(program: commander.CommanderStatic) {
   command.option('-c, --config <file>', `Configuration file, or a test directory with optional "${tsConfig}"/"${jsConfig}"`);
   command.option('--forbid-only', `Fail if test.only is called (default: false)`);
   command.option('-g, --grep <grep>', `Only run tests matching this regular expression (default: ".*")`);
+  command.option('-gTL, --grepTestLocations <grep>', `Only run tests by the file location. Comma separated values in the form of line:column.`);
```

Comment:
I would not do that, i'd rather accept plain argument in the form of page.spec.ts:45